src/components/ServiceRequestForm.jsx

```diff
@@ -455,7 +455,7 @@ const ServiceRequestForm = (props, errors, touched) => {
 				<SexType
 					shouldShow={isAnimalCategory}
 					requestType={requestType}
-					returnRequestTypes={returnRequestTypes("requestType_petAndAnimalIssue")}
+					returnRequestTypes={returnConfigItems("formTypes","requestType_petAndAnimalIssue")}
```

Comment:
you should set each of these config items for animals into a variable since they are repeated.

- "requestType_petAndAnimalIssue"
- "petTypeCat"
- "petTypeDog"

i suspect there could be some other improvements, but in the scope of this change, that would all be required.

Reply:
Fixed